src/panel/TemplateEditor/DataPanel/PortalView.tsx

```diff
@@ -15,14 +16,31 @@ export default () => {
         template: templateFile,
         url: inspectedWindowUrl
     }).toString();
-    const url = `${portal?.views?.panel}?${params}`;
-    const seller_id = !!template?.obj?.params?.seller_id ? String(template?.obj?.params?.seller_id) : '';
-    const tempUrl = `${portal?.views?.panel}&hide_filter=Retailer+ID,Seller+ID`.replace(/seller_id/g, seller_id);
+    const [ embedUrl, setEmbedUrl ] = useState<string>();
+
+    useEffect(() => {
```

Comment:
Move this into a new PortalContext within the TemplateEditor context such that `panelUrl` comes from `usePortal`.